tests/sitemaps_tests/test_http.py

```diff
@@ -27,6 +27,25 @@ def test_simple_sitemap_index(self):
 """ % self.base_url
         self.assertXMLEqual(response.content.decode(), expected_content)
 
+    def test_simple_sitemap_index_not_callable(self):
+        "A simple sitemap index can be rendered"
```

Comment:
If this is exactly the same as the above test, was the goal to add a new test or should this be removed for the time being?